gms_process_inputs/rnaseq.process_inputs.pl

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env genome-perl
+
+use strict;
+use warnings;
+
+use above 'Genome';
+
+use File::Spec;
+use YAML::XS;
+
+my $build_id = $ARGV[0]
+    or die 'no build id';
+
+my $build = Genome::Model::Build->get($build_id)
+    or die 'no build for id';
+
+my @inputs = $build->inputs;
+
+my %inputs;
+for my $file_input ('reference_index', 'trimming_adapters', 'reference_annotation', 'kallisto_index', 'gene_transcript_lookup_table','refFlat','ribosomal_intervals') {
```

Comment:
It'd be better if the inputs here used the `InputProcessor` strategy where possible like in the `somatic_exome.process_inputs.pl` in this PR.

Reply:
Since this is the latest version of the process script currently in the pipeline, I think I'm going to leave it here and punt on doing this for now. Next time we update the PP, I completely agree that we should change it to use that strategy.